recipes/pycifrw/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "PyCifRW" %}
+{% set version = "4.3" %}
+{% set sha256 = "3b19e80278b40ebe00ad49a7dd8813007fbdc7a9a43fd3377e404ac0bcbcd3c3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - CifFile.CifFile
+    - CifFile.StarFile
+    - CifFile.StarScan
+
+about:
+  home: https://bitbucket.org/jamesrhester/pycifrw/overview
+  license: Python2
```

Comment:
Normally this would PSF 2, but they changed out the names. Given that we probably should say `PSF-2-like`.